awx/main/conf.py

```diff
@@ -334,6 +334,19 @@
     category_slug='jobs',
 )
 
+register(
+    'AWX_MOUNT_ISOLATED_PATHS_ON_K8S',
```

Comment:
Is this _just_ on K8s?

Can't traditional installs also make use of container group jobs? Would this be usable in those situations as well?

Reply:
This should work with container groups too. Maybe let's rename it to make more inclusive

Reply:
At the tooltip, you can see the user will see the `Container Groups` text. Since this feature works for K8S installs and traditional installs with Container Groups, do you think we should rename it or doc string already should be sufficient?

Reply:
yeah, that works